server/container_start.go

```diff
@@ -34,8 +36,20 @@ func (s *Server) StartContainer(ctx context.Context, req *pb.StartContainerReque
 		if err := s.ContainerStateToDisk(c); err != nil {
 			log.Warnf(ctx, "unable to write containers %s state to disk: %v", c.ID(), err)
 		}
+
+		if strings.Contains(runtimeHandler, runtimeHandlerHighPerformance) && shouldCPULoadBalancingBeDisabled(c.Annotations()) {
```

Comment:
nit: since we do this operation twice, can we save whether the runtimeHandler contains runtimeHandlerHighPerformance in a variable above?

Reply:
sure will do it

Reply:
done